compose/mpp/demo-uikit/build.gradle.kts

```diff
@@ -19,39 +21,44 @@ repositories {
 }
 
 kotlin {
-    iosX64("uikitX64") {
-        binaries {
-            framework {
-                baseName = "shared"
-                freeCompilerArgs += listOf(
-                    "-linker-option", "-framework", "-linker-option", "Metal",
-                    "-linker-option", "-framework", "-linker-option", "CoreText",
-                    "-linker-option", "-framework", "-linker-option", "CoreGraphics"
-                )
+    if (System.getProperty("os.arch") == "aarch64") {
+        iosSimulatorArm64("uikitSimArm64") {
```

Comment:
Does it cause some issue when `iosSimulatorArm64("uikitSimArm64")` is declared unconditionally?